Sources/KlaviyoLocation/KlaviyoSDK+Location.swift

```diff
@@ -16,11 +16,15 @@ extension KlaviyoSDK {
     public func registerGeofencing() {
         Task {
             await MainActor.run {
-                KlaviyoLocationManager.shared.setupGeofencing()
+                KlaviyoLocationManager.shared.startGeofenceMonitoring()
             }
         }
     }
 
+    // Q: have some questions about this?
+    // what code in the manager init does this actually need?
+    // if we know what can we just call that instead?
+    // I still don't think I fully understand why this is needed?
     /// To be called in didFinishLaunchingWithOptions to ensure geofence events that happen in a backgrounded/terminated state are processed.
     public func monitorGeofencesFromBackground() {
         _ = KlaviyoLocationManager.shared
```

Comment:
@belleklaviyo to dig into this a bit and also may be a create a internal method in `KlaviyoLocationManager` that is specific to `monitorGeofencesFromBackground` and this method will also be called from `KlaviyoLocationManager.init()`.